src/hyperion/external_interaction/callbacks/xray_centre/ispyb_callback.py

```diff
@@ -163,11 +162,20 @@ def activity_gated_event(self, doc: Event):
 
         return doc
 
+    def populate_axis_info_for_snapshot(
+        self, data_collection_info: DataCollectionInfo, omega_start: float | None
+    ):
+        if omega_start is not None:
+            data_collection_info.omega_start = omega_start
+            data_collection_info.axis_start = omega_start
+            data_collection_info.axis_end = omega_start
+            data_collection_info.axis_range = 0
```

Comment:
It's got the name because it is an implementation of an abstract method which is called from the base class `_handle_oav_snapshot_triggered` method - so the method definitely is snapshot-specific.